src/main/java/seedu/academyassist/model/AcademyAssist.java

```diff
@@ -18,6 +19,7 @@
 public class AcademyAssist implements ReadOnlyAcademyAssist {
 
     private final UniquePersonList persons;
+    private int studentCount;
```

Comment:
Maybe this could be a static field? Since we only going to have one instance for Academy Assist to track the studentCount.

Reply:
Actually I have thought about it. What makes me keep it as a private instance variable is to prevent unintended changes (which could lead to unwanted side effects). Restricting changes to studentCount through incrementStudentCount() or setStudentCount() gives us better control. 

I'm not sure if this will bring unnecessary overhead tho.. Any thought whether it is better still to make it static?